cms/templates/cms/program_page.html

```diff
@@ -11,7 +12,8 @@
   <div class="app-media layout-boxed">
     <div class="main-content-wrapper">
       <section class="main-content">
-        <div class="banner-wrapper banner-sub banner-program-1">
+      {% image page.background_image fill-1310x613 as tmp_photo %}
```

Comment:
1. just curious: what is this `image` function and `fill-1310x613` argument accomplishing that can't be done with CSS?
1. can you rename `tmp_photo` to something more descriptive? i think simply renaming it `background_image` would work